### ISSUE TITLE
Enable rebalancing channels

### ISSUE THREAD

**Issue by bhide-fr** [LN-Zap/zap-desktop#1198]
Can channel rebalancing (using the sendtoroute command in lnd I suppose) be added to zap? 

My first github issue / feature request, please advise on appropriate formatting and etiquette of these things :)

**Comment by mrfelton** [comment]
@bhide-fr I would definitely like to see channel rebalancing features added to Zap.

Given that this type of change would probably require some changes/additions to the UI we will want to consider how this would be presented to users.

We do already have a redesign of the entire channel manage piece in mind, but I'm not sure of the current state of this. @JimmyMow do you know if this is something that has been looked at all yet from a design perspective?

@bhide-fr  as for contributing guidelines - take a look at https://github.com/LN-Zap/zap-desktop/blob/master/CONTRIBUTING.md first.

**Comment by JimmyMow** [comment]
@mrfelton yes I plan on incorporating this into our new channel UI. It's unclear whether it is outside the scope of new channel UI v1, but this along with things like submarine swaps should be added in the future

**Comment by ZapUser77** [comment]
I have a PR to add it internally to LND: 
https://github.com/lightningnetwork/lnd/pull/1751
https://github.com/lightningnetwork/lnd/pull/1759

And there is currently a python script being developed to do this externally:
https://github.com/C-Otto/rebalance-lnd

Hope this helps.  ^_^
